new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiConfigMapOverride.java

```diff
@@ -0,0 +1,669 @@
+    } catch (java.lang.InterruptedException ie) {
+      logger.info("Got InterruptedException during Thread.sleep");
+      fail("Got InterruptedException during Thread.sleep");
+    }
+    // Verify a rolling restart is triggered in a sequential fashion 
+    // admin-server --> managed-server1 --> managed-server2 
+    logger.info("Check admin server pod {0} to be restarted in namespace {1}",
+        adminServerPodName, domainNamespace);
+    checkPodDeleted(adminServerPodName);
+    checkPodCreated(adminServerPodName);
+    checkPodRunning(adminServerPodName);
+    checkServiceCreated(adminServerPodName);
+
+    // check managed server pods re-started sequentially
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Wait for managed server pod {0} to be restarted in namespace {1}",
+                        managedServerPrefix + i, domainNamespace);
+      checkPodDeleted(managedServerPrefix + i);
+      checkPodRunning(managedServerPrefix + i);
+      checkServiceCreated(managedServerPrefix + i);
+    }
+
+    oracle.weblogic.kubernetes.utils.ExecResult result = null; 
+    try {
+      String hostname = null;
+      if (System.getenv("K8S_NODEPORT_HOST") == null) {
+        hostname = InetAddress.getLocalHost().getHostName();
+      } else {
+        hostname = System.getenv("K8S_NODEPORT_HOST");
+      }
+      checkJdbc =  new StringBuffer("status=$(curl --user weblogic:welcome1 ");
+      checkJdbc.append("http://" + hostname + ":30711")
+             .append("/management/wls/latest/datasources/id/TestDataSource/")
+             .append(" -o /dev/null")
+             .append(" -w %{http_code});")
+             .append("echo ${status}");      
+      logger.info("CURL command {0}", new String(checkJdbc));
+      result = exec(new String(checkJdbc),true);
+    } catch (java.net.UnknownHostException uhe) {
+      logger.info("UnknownHostException is received {0}", uhe);
+      fail("Got UnknownHostException");
+    } catch (Exception ex) {
+      logger.info("Unexpected exception  {0}", ex);
+      fail("Got unexpected exception");
+    }
+    logger.info("Curl command returns {0}", result.toString());
+    assertEquals("200",result.stdout(),"Datasource configuration not found");
+    logger.info("Found the DataSource configuration ");
+  }
+
+  @AfterEach
+  public void tearDown() {
+
+    // Delete domain custom resource
+    logger.info("Delete domain custom resource in namespace {0}", domainNamespace);
+    assertDoesNotThrow(() -> deleteDomainCustomResource(domainUid, domainNamespace),
+        "deleteDomainCustomResource failed with ApiException");
+    logger.info("Deleted Domain Custom Resource " + domainUid + " from " + domainNamespace);
+
+    // delete the domain image created for the test
+    if (miiImage != null) {
+      deleteImage(miiImage);
+    }
+
+  }
+
+  /**
+   * Uninstall Operator, delete service account, domain namespace and
+   * operator namespace.
+   */
+  @AfterAll
+  public void tearDownAll() {
```

Comment:
don't need this - extension will do it

Reply:
I will update once the extension is available in the develop branch. As per my discussion with @sankarpn his PR is not merged into develop

Reply:
done